source/unified-test-format/unified-test-format.rst

```diff
@@ -3898,6 +3898,8 @@ Changelog
 
 ..
   Please note schema version bumps in changelog entries where applicable.
+:2022-11-16: Update tests for ``observeSensitiveCommands`` that use ``getnonce`` to
+             only run on server versions < 6.2.
```

Comment:
```suggestion
```

Tests shouldn't need changelog entries in the main spec. I don't recall this ever being done before, although some test directories do have a README file with a changelog (mostly for prose tests).

Reply:
sgtm. there was some precedent for describing test changes in the CLAM spec changelog, but happy to remove it from the changelog here